owasp-suppressions.xml

```diff
@@ -0,0 +1,111 @@
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
+  <suppress>
+    <notes><![CDATA[
+   file name: grpc-client-utils-0.12.0.jar
+   file name: grpc-context-utils-0.12.0.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.hypertrace\.core\.grpcutils/.*@.*$</packageUrl>
+    <cpe>cpe:/a:grpc:grpc</cpe>
+  </suppress>
+
+  <suppress>
+    <notes><![CDATA[
+   file name: grpc-context-utils-0.12.0.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.hypertrace\.core\.grpcutils/grpc\-context\-utils@.*$</packageUrl>
+    <cpe>cpe:/a:utils_project:utils</cpe>
+  </suppress>
+
+  <suppress>
+    <notes><![CDATA[
+   file name: pinot-avro-0.12.1.jar
+   file name: pinot-avro-base-0.12.1.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.apache\.pinot/pinot\-avro.*@.*$</packageUrl>
+    <cpe>cpe:/a:apache:avro</cpe>
+  </suppress>
+
+  <suppress>
+    <notes><![CDATA[
+   file name: pinot-minion-builtin-tasks-0.12.1.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.apache\.pinot/pinot\-minion\-builtin\-tasks@.*$</packageUrl>
+    <cpe>cpe:/a:tasks:tasks</cpe>
+  </suppress>
+
+  <suppress>
+    <notes><![CDATA[
+   file name: pinot-thrift-0.12.1.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.apache\.pinot/pinot\-thrift@.*$</packageUrl>
+    <cpe>cpe:/a:apache:thrift</cpe>
+  </suppress>
+
+  <!-- Disputed vulnerability -->
+  <suppress>
+    <notes><![CDATA[
+   file name: jackson-databind-2.14.2.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/com\.fasterxml\.jackson\.core/jackson\-databind@.*$</packageUrl>
+    <cve>CVE-2023-35116</cve>
+  </suppress>
+
+  <suppress>
+    <notes><![CDATA[
+   file name: calcite-babel-1.29.0.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.apache\.calcite/calcite\-babel@.*$</packageUrl>
+    <cpe>cpe:/a:apache:calcite</cpe>
+  </suppress>
+
+  <suppress>
+    <notes><![CDATA[
+   file name: kotlin-stdlib-1.4.0.jar
+   file name: kotlin-stdlib-jdk7-1.3.71.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.jetbrains\.kotlin/kotlin\-stdlib.*$</packageUrl>
+    <cpe>cpe:/a:jetbrains:kotlin</cpe>
+  </suppress>
+
+  <suppress>
+    <notes><![CDATA[
+   file name: spark-tags_2.12-3.2.1.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.apache\.spark/spark\-tags_2\.12@.*$</packageUrl>
+    <cpe>cpe:/a:apache:spark</cpe>
+  </suppress>
+
+  <suppress>
+    <notes><![CDATA[
+   file name: wire-schema-3.2.2.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/com\.squareup\.wire/wire\-schema@.*$</packageUrl>
+    <cpe>cpe:/a:wire:wire</cpe>
+  </suppress>
+
+  <!-- This is the final version, and its coming via transitive dep. -->
+  <suppress>
+    <notes><![CDATA[
+   file name: commons-httpclient-3.1.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/commons\-httpclient/commons\-httpclient@.*$</packageUrl>
```

Comment:
Similar to existing comment thread, we're suppressing true positives here.

Reply:
Added specific CVE. No upgrade for this version.

Reply:
Ugh - I can't believe pinot is using a 16 year old http client, this was released in 2007. We really need to get them on to take care of their dependencies. Specific CVE with an expiration would be the right approach here.